_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,288 @@
+| 4.   | [LGPL v2.1+][LGPL v2.1]             | [LGPL v2.1+][LGPL v2.1]                                                   | [LGPL v2.1+][LGPL v2.1]                                           | Once Apache 2.0 dependencies have been removed from the library.                      | N/A                 |
+
+In this section, we will briefly go over the various steps we propose to take (Table 2) as we go about relicensing the
+[MDAnalysis core library][].
+
+### 1. A GPL v2+ package and source code (MDAnalysis v2.5 and lower)
+
+This is the original state of the license up until the [2.5.x releases of MDAnalysis](https://www.mdanalysis.org/blog/page2/#releases-2-5-0-and-2-4-x-of-mdanalysis).
+The package and source code were released under a [GPL v2+][GPL v2] license.
+
+### 2. A GPL v3+ package, and a mix of historical GPL v2+ and new LGPL v2.1+ source code (MDAnalysis v2.6 onwards)
+
+As of commit [44733fc][], all new contributions to MDAnalysis have been made under the terms of the
+[LGPL v2.1+][LGPL v2.1] and the package is released under [GPL v3+][GPL v3] to better reflect our
+understanding of the [Apache 2.0 incompatibility issues](#why-the-interim-gpl-v3-and-lgpl-v3-package-licenses-the-issue-with-apache-20).
+Historical source code contributions remain under the terms of [GPL v2+][GPL v2].
+
+This will remain the state of MDAnalysis until we obtain approval from our developers to relicense
+their code to [LGPL v2.1+][LGPL v2.1]. As we obtain contributor agreements, these historical
+source code contributions will slowly turn to [LGPL v2.1+][LGPL v2.1].
+
+### 3. A LGPL v3+ package, and an LGPL v2.1+ source code (Optional)
+
+Should we obtain approval to relicense from our developers before we remove our [Apache 2.0][]
+dependencies, we will then start releasing the MDAnalysis package under a [LGPL v3+][LGPL v3] license.
+
+### 4. An LGPL v2.1+ package and source code
+
+This is the state which we aim the library to eventually be licensed under. Once we both obtain
+the agreement of our developers to relicense and remove [Apache 2.0][] dependencies, we will be
+able to release the MDAnalysis package under [LGPL v2.1+][].
+
+## How does this affect me?
+
+### As a contributor
+
+If you have any historical contributions you will be asked to relicense them to [LGPL v2.1+][LGPL v2.1]
+instead of the initially proposed [LGPL v3+][LGPL v3].
+
+### As a developer
+
+Ultimately we believe that the proposed change to [LGPL v2.1+][LGPL v2.1] should offer a better compatibility
+(Table 1) with whatever license you may choose to use for your own code. The interim package license changes
+may however impact you, if you believe this to be the case [please do get in touch with us](#getting-in-touch).
+
+### As a user
+
+We do not anticipate this to have a significant impact on users. There may be some slight changes to how you
```

Comment:
How? I think I mentioned this before, but we might take inspiration from https://github.com/spack/spack/issues/9144 (if we haven't already).

Reply:
More specifically: https://github.com/spack/spack/issues/9137

Reply:
I think this comment is linking to the wrong place?

Reply:
Oh, shit. Looks like all of them are off...

Bug in the GitHub iOS app, most likely...

Reply:
I got the idea of most of them :)

Reply:
This was meant for the section below:

> We will shortly be contacting every historical contributor to MDAnalysis and ask them to agree to changing
their contribution’s license from the current “GPL v2 or any later version” to “LGPL v2.1 or any later version”.

Really sorry for the confusion.